src/scripts/generate-form.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# This simple CGI script helps create a tree browser for ONTIE
+
+cd ../..
+
+URL="http://example.com?${QUERY_STRING}"
+TEMPLATE=$(urlp --query --query_field=template "${URL}")
+ADD=$(urlp --query --query_field=add "${URL}")
+BRANCH=$(git branch --show-current)
+
+MESSAGE="None"
+if [[ ${ADD} ]]; then
+	QS=$(urlp --query "${URL}")
+	MESSAGE=$(python3 src/scripts/add-term.py "${QS}")
```

Comment:
@jamesaoverton if this python script fails, this just returns an empty message. Do you know if there is any way to make a non-zero exit code break here so that we can see the exception message? I can check for an empty message, but then we lose the exception and don't know why it failed.

Reply:
When I check this branch out in DROID and go <https://droid.ontodev.com/ONTIE/branches/template-forms/views/src/scripts/generate-form.sh?template=assays> I get this, saying `urlp` is not installed:

<img width="1366" alt="droid_cgi_urlp" src="https://user-images.githubusercontent.com/292409/120647624-cebb4180-c448-11eb-8e65-070b42db2f26.png">

Reply:
It's added to requirements now